lib/onfido/configuration.rb

```diff
@@ -1,45 +0,0 @@
-# frozen_string_literal: true
-
-module Onfido
-  module Configuration
-    REGION_HOSTS = {
-      us: 'api.us.onfido.com',
-      ca: 'api.ca.onfido.com'
-    }.freeze
-
-    attr_accessor :api_key, :region, :open_timeout, :read_timeout
-
-    def self.extended(base)
-      base.reset
-    end
-
-    def configure
-      yield self
-    end
-
-    def reset
-      self.api_key = nil
-      self.region = nil
-      self.open_timeout = 30
-      self.read_timeout = 80
-      RestClient.log = nil
-    end
-
-    def logger=(log)
```

Comment:
Configuring logger removed - this just set the logger for the whole of RestClient, so would have affected requests made outside of onfido-ruby.